scripts/start.sh

```diff
@@ -58,6 +58,10 @@ gzip -f -9 -k /www/index.html
 # gzip metadata.json again because we changed it
 gzip -f -9 -k /www/metadata.json
 
+# Create file to include into nginx config to use
+# the pods nameserver as resolver
+echo resolver $(awk '/^nameserver/{print $2}' /etc/resolv.conf) ";" > /etc/nginx/config/resolvers.conf
```

Comment:
Just to double check -- it's still valid nginx syntax if there are multiple entries here?

Reply:
Didn't think of that. I'll experiment and modify accordingly if neccessary.

Reply:
Turns out this was working as is, I've disabled ipv6 resolving because I am unsure if our clusters support ipv6 yet.